specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2022-04-01-preview/apimdeployment.json

```diff
@@ -1086,6 +1086,19 @@
           "description": "Gateway URL of the API Management service in the Region.",
           "readOnly": true
         },
+        "enableNATGateway": {
+          "type": "boolean",
+          "description": "Property only meant to be used for Premium SKU Service. This can be used to enable NAT Gateway.",
+          "default": false
+        },
```

Comment:
Consider if you can use enum for this. see [https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html](https://armwiki.azurewebsites.net/rp_onboarding/process/api_review_best_practices.html)

Reply:
changes applied